addons/hr_recruitment/models/hr_applicant.py

```diff
@@ -493,6 +498,57 @@ def name_get(self):
                 for applicant in self
             ]
         return super().name_get()
+    @staticmethod
```

Comment:
The parsing methods could be moved to a seperate file (like parsers.py) and simply be imported from there